src/controllers/touristCenterController.js

```diff
@@ -0,0 +1,120 @@
+import db from "../services/AdminServices/touristCenterService";
+import { validation, validateId } from "../validation/touristCenterValidation";
+
+/**
+ * @class touristCenterController
+ * @description create, read, update & delete touristcenter
+ * @exports touristCenterController
+ */
+export default class touristCenterController {
+  /**
+   * @param {object} req - The user request object
+   * @param {object} res - The user response object
+   * @returns {object} Success message
+   */
+  static async addTouristCenter(req, res) {
+    try {
+      const {
+        countryId, gallery, name, location, about
+      } = req.body;
+      const { error } = validation(req.body);
```

Comment:
remove controller at the end of the controller name, for it to be in the controllers' folder, we already know its a. controller.